German translation of django.po and django.mo

horshack opened the pull request:
I took the french translation and translated every string mentioned via poeditor.com. I made no further translations.

cuu508 commented:
Thanks for the translations, @horshack !

I moved the translation files under LC_MESSAGES directory so Django can find them, and fixed `%(site_name)`  -> `%(site_name)s` template syntax. Here's how the front page looks:

![image](https://user-images.githubusercontent.com/661859/119105695-425f5680-ba26-11eb-9581-4dcab751a69f.png)

If all looks good, I'm happy to merge it.

horshack commented:
The translation looks good! Thanks for your great work with 
healthchecks.
Greetings from Bavaria,
Richard

------ Originalnachricht ------
Von: "Pēteris Caune" ***@***.***>
An: "healthchecks/healthchecks" ***@***.***>
Cc: "Richard Lippmann" ***@***.***>; "Mention" 
***@***.***>
Gesendet: 21.05.2021 10:19:57
Betreff: Re: [healthchecks/healthchecks] German translation of django.po 
and django.mo (#514)

>
>Thanks for the translations, @horshack <https://github.com/horshack> !
>
>I moved the translation files under LC_MESSAGES directory so Django can 
>find them, and fixed %(site_name) -> %(site_name)s template syntax. 
>Here's how the front page looks:
>
>image
><https://user-images.githubusercontent.com/661859/119105695-425f5680-ba26-11eb-9581-4dcab751a69f.png>
>
>If all looks good, I'm happy to merge it.
>
>—
>You are receiving this because you were mentioned.
>Reply to this email directly, view it on GitHub 
><https://github.com/healthchecks/healthchecks/pull/514#issuecomment-845757970>, 
>or unsubscribe 
><https://github.com/notifications/unsubscribe-auth/AABI3TFMBRIGDVRIH4PBCETTOYJS3ANCNFSM446G7DCQ>.
>
